src/generate_module.ts

```diff
@@ -0,0 +1,81 @@
+/// <reference path="../typings/node/node.d.ts"/>
+/// <reference path="../typings/xml2js/xml2js.d.ts"/>
+/// <reference path="../typings/es6-promise/es6-promise.d.ts"/>
+/// <reference path="./parsers.ts"/>
+/// <reference path="./generators.ts"/>
```

Comment:
You don't need last two references if you're importing modules explicitly (like `import parsers = require("parsers");` below).